components/todo/TodoContainer.vue

```diff
@@ -61,6 +63,7 @@ export default {
   methods: {
     onAddTodo(todo) {
       this.$store.dispatch('todos/add', todo)
+      toast('success', this.$t('added', { item: this.$t('todo') }))
```

Comment:
this line indicates the toaset shows the todo name in the message when adding the todo. But from my local test and also from the video, it seems like thats not the case, a generic message `Todo successfully added` is shown. could you please have a look?

Reply:
@bia-wtag Bhai, It's not todo name, it's the "Todo" which is in a variable, suppose, we we added a job, then the code will be, toast('success', this.$t('added', { item: this.`$t('job')` }))
Thanks